### PR TITLE
Add credentials to image fetch

### PR DESCRIPTION
**Pull request by mariarek**
Add property 'crossorigin' to Image, to allow credentials to be passed with the request.

### CODE REVIEW

**Comment by mariarek** [comment]
@berickson1 That's exactly the issue - the origin header is not being set by the browser at all for the  / background-image .
Browser only sends the origin header for  if the `crossorigin=use-credentials` attribute is there - but there is nothing like that for the background-image.

File: src/common/Types.ts [hunk 1]
```diff
@@ -529,6 +529,8 @@ export interface ImagePropsShared extends CommonProps {
     onError?: (err?: Error) => void;
 
     shouldRasterizeIOS?: boolean; // iOS-only prop, if view should be rendered as a bitmap before compositing
+
+    crossorigin?: boolean;
```
[the remark this answers is not in the document]
Comment (mariarek) [reply src/common/Types.ts]: Updated the docs, but I don't really know how to make the test work - I see that the images used in the test app are hosted on https://microsoft.github.io/reactxp/img/tests - I would need to have a way to only accept this specific combo of cookie + origin header...

**Comment by erictraut** [comment]
Would it make sense to always assume crossorigin if there's an "origin" header present in the request? We could document that behavior. It would be web-only, of course. I'd prefer to find a way to do this without exposing another (platform-specific) prop if possible.

**Comment by mariarek** [comment]
@erictraut I'm fine with that solution - should it be documented somewhere else, besides the code?

**Comment by erictraut** [comment]
OK, let's go with that approach. It's fine to document it only in the code in this case. It's sufficiently obscure that I don't think other users will care about it.

**Comment by mariarek** [comment]
@erictraut is the current version okay then? :)

**Pull request merged**